core/templates/dev/head/pages/exploration_player/ProgressNavDirective.js

```diff
@@ -47,13 +68,12 @@ oppia.directive('progressNav', [
             }
           };
 
-          $scope.hasPrevious = function() {
-            return $scope.currentCardIndex > 0;
-          };
-
-          $scope.hasNext = function() {
-            return $scope.currentCardIndex < $scope.transcriptLength - 1;
-          };
+          $scope.showContinueButton = function() {
```

Comment:
Seems more like $scope.isContinueButtonShown (since it returns a boolean)

Reply:
changed to `shouldContinueButtonBeShown`.